rskj-core/src/test/java/co/rsk/peg/BridgeUtilsTest.java

```diff
@@ -2092,22 +2092,32 @@ public void extractHash160FromBytes() throws BridgeIllegalArgumentException {
 
     @Test
     public void calculatePegoutTxSize() {
-        Federation fed = new Federation(
-            Arrays.asList(FederationMember.getFederationMemberFromKey(new BtcECKey())),
-            Instant.now(),
-            0,
-            networkParameters
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP271)).thenReturn(true);
+
+        List<BtcECKey> keys = createBtcECKeys(13);
+        Federation federation = new Federation(
+                FederationMember.getFederationMembersFromKeys(keys),
+                Instant.now(),
+                0,
+                networkParameters
         );
 
-        int pegoutTxSize = BridgeUtils.calculatePegoutTxSize(fed, 1, 1, 1, 1);
+        int pegoutTxSize = BridgeUtils.calculatePegoutTxSize(activations, federation, 2, 2);
+
+        // The difference between the calculated size and a real tx size should be smaller than 1% in any direction
+        int origTxSize = 2076; // Data for 2 inputs, 2 outputs Based on Pegouts From Blockchain.info Explorer
+        int difference = origTxSize - pegoutTxSize;
+        double tolerance = origTxSize * .01;
 
-        // 58 accounts for all the added values as part of a peg-out tx
-        int calculation = fed.getRedeemScript().getProgram().length+ 58;
-        assertEquals(calculation, pegoutTxSize);
+        assertTrue(difference < tolerance && difference > -tolerance);
     }
```

Comment:
Let's add tests for other transactions sizes. 

- 3 inputs and 2 outputs
- 4 inputs, etc
- Also multiple outputs since we expect to start batching pegouts

Reply:
Done